src/wp-includes/load.php

```diff
@@ -753,7 +753,7 @@ function wp_start_object_cache() {
 			)
 		);
 
-		wp_cache_add_non_persistent_groups( array( 'counts', 'plugins' ) );
+		wp_cache_add_non_persistent_groups( array( 'counts', 'plugins', 'theme_json' ) );
```

Comment:
This is also done at https://github.com/WordPress/wordpress-develop/pull/3556 and https://github.com/WordPress/wordpress-develop/pull/3712 If any of them land first, this won't be needed when merging this PR.

Reply:
This line of code needs  to be removed from this PR as it was covered in PR #3556, which has now been committed via https://core.trac.wordpress.org/changeset/55086. I'll see if I can rebase and update the PR 🤞